### ISSUE TITLE
Generate directory listing

### ISSUE THREAD

**Issue by ahri** [benweet/stackedit#1204]
I'm not sure if this is a feature request or existing functionality I'm not aware of:

1. I'd like to write blogs with StackEdit, apply metadata to them which I can display in a template. I know how to do this already.

2. Then from a directory containing those blog entries, I'd like to generate an index for them, using the metadata to fish out date/title/etc.

3. I guess this might be a wider piece of functionality to publish a single file from a collection of other files, with the iteration happening in a template. Individual pages for each tag might be another use-case.

4. It might also be good to be able to apply (3) to "all files with metadata matching x".

**Comment by benweet** [comment]
Indeed, the plan is to be able to apply a template to multiple files (that's why you have an array of files passed in the template). Obviously, the main use case will be to produce an html from files in a folder.
However, there is no plan to label files and have filters based on labels. It would affect the simplicity of the app I believe.

**Comment by ahri** [comment]
That's great news :)

I understand your point about simplicity, I hoped that point (4) might be simple enough, or hidden in an "advanced" menu or somesuch.

Your call of course, thanks a lot for a great app!
